BraintreeUIKit/Models/BTUIKCardType.m

```diff
@@ -134,37 +111,37 @@ + (NSArray *)allCards
 
         BTUIKCardType *visa = [[BTUIKCardType alloc] initWithBrand:BTUIKLocalizedString(CARD_TYPE_VISA)
                                                   securityCodeName:BTUIKLocalizedString(CVV_FIELD_PLACEHOLDER)
-                                                          prefixes:@[@"4"]];
+                                                          prefixes:@[@"^4\\d*"]];
```

Comment:
I might consider removing or deprecating `validNumberPrefixes` in favor of `binRangePattern` which can be the regex pattern string.